client/blocks/login/two-factor-authentication/verification-code-form.scss

```diff
@@ -100,3 +113,14 @@
 .security-key-form__add-wait-for-key-heading {
 	font-weight: 600;
 }
+
+div.two-factor-auth-enabled:not(.woo) {
+	.login__form-header {
+		width: 352px;
```

Comment:
@jameskoster @matt-west do we have/should we establish some token-widths so they can be more universal?

Reply:
Yes, it would be great to have these defined somewhere. I’m not aware of any at the moment, but some are emerging from the Forms & Settings atelier.

Reply:
Fixed in b7fafdcd9a61023d70a1ba3c80a90f1c0a3236e4!